### PR TITLE
imgui: update to 1.88

### PR DESCRIPTION
**Pull request by neheb**
Fixed compilation under MinGW.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

**Comment by neheb** [comment]
ping @stephanlachnit

File: ci_config.json [hunk 1]
```diff
@@ -159,14 +159,6 @@
       "gettext"
     ]
   },
-  "imgui": {
-    "debian_packages": [
```
Comment (stephanlachnit) [inline ci_config.json:163]: Why are these removed?

Comment (neheb) [reply ci_config.json:163]: Not needed.

Comment (stephanlachnit) [reply ci_config.json:163]: Ehm, yes but they provide a useful test* for the CI? Please add them back...

* i.e. that the dependency lookup and compilation of the files still work

Comment (neheb) [reply ci_config.json:163]: IMO It's more useful to test other wraps.

Maybe edit CI to also test --wrap-mode=forcefallback

Comment (stephanlachnit) [reply ci_config.json:163]: Ah, fair, glfw and sdl2 are also available as wrap. allegro5 is not though.
